src/ocean/io/Path.d

```diff
@@ -1460,11 +1462,11 @@ in
             switch (pattern[i])
                    {
                    case '[':
-                        assert(!inbracket);
```

Comment:
I've grepped for `in$` and this showed up as a remaining `in` contract. This feels very expensive to be left in `release` build. Should I just leave it here and use `assert`?

Reply:
Is `patternMatch` a function that you could imagine being called in time critical code?

Reply:
Puh, I guess you're right. It's still `O(n)`, but let's KISS.